etc/c/curl.d

```diff
@@ -1601,7 +1601,7 @@ struct curl_slist
  * Appends a string to a linked list. If no list exists, it will be created
  * first. Returns the new list, after appending.
  */
-curl_slist * curl_slist_append(curl_slist *, in char *);
```

Comment:
should I replace them back with in? I thought it would be a more precise consistency if we used just const